powerhome/playbook
# Update Dialog id prop

#### Screens

[INSERT SCREENSHOT]

#### Breaking Changes

No, moving id prop from dialog container to modal

#### Runway Ticket URL

#### How to test this
Check to see if kit appears on the docs page
Add an id to dialog kit
inspect the dialog kit to see that an id appears on the modal

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
